driver.go

```diff
@@ -127,6 +131,12 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Usage: `VM instance size to create ("g3-standard-0.25-kvm", "g3-standard-0.5-kvm", etc)`,
 			Value: defaultTritonPackage,
 		},
+		mcnflag.StringFlag{
+			EnvVar: "OS_SSH_USER",
+			Name:   "triton-ssh-user",
```

Comment:
If this PR ends up being something @misterbisson is +1 on, these will need to be adjusted to use `flagPrefix` and `envPrefix` just like all the other flags. :wink:

Reply:
@tianon Fixed! Can't believe I missed that 😄